### PR TITLE
(multiple) better assault rifle description

### PR DESCRIPTION
**Pull request by Duke-M-commits**
After reading about the description of the assault rifle being a bit confusing on [Discord](https://discord.com/channels/310381338538541056/310381338538541056/1304246470903337040) i went to fix it.

I've also machine-retranslated it, but since we do not have active translators for all languages I went and indented the ones for which there is no one to look after (danish, hungarian, polish, portuguese, spanish, turkish).

Please review:
brazilian @Mart-User or @FurtadoPires
~~finnish @Crazymanbos~~
french @wawawawawawawa
~~italian @MakinDay or @Steffo99~~
~~japanese @Deerlord0 or @finishtw~~
~~korean @hangumi or @BRAIN-IS-BACK or @906bc906~~
latam @gaelcoral
russian @Blueberryy
~~schinese @ModdedMarionette or @ywgATustcbbs or @zhongqingsong~~
~~swedish @Crazymanbos~~
~~tchinese @ModdedMarionette or @Crazy3lf or @mewindng~~
ukrainian @MsSockol
~~vietnamese @DucDuyGaming~~

PS: you can suggest a change to any line like this
image

### CODE REVIEW

File: resource/reactivedrop_italian.txt [hunk 1]
```diff
@@ -2405,8 +2405,8 @@
...
-"asw_wdesc_rifle"		"Il 22A3-1 è l'arma di combattimento standard in dotazione ai soldati delle Forze Armate Interstellari (IAF). Offrendo un'elevata cadenza di tiro, proiettili esplosivi ad alta velocità e un lanciagranate integrato, quest'arma garantisce una notevole versatilità sul campo, ed è capace di abbattere sia bersagli piccoli che grandi. La dimensione standard del caricatore è di 98 colpi."
+"[english]asw_wdesc_rifle"		"The 22A3-1 is the standard issue combat weapon given to an Interstellar Armed Forces marine. Offering high rate of fire and high velocity explosive rounds for the underslung grenade launcher, this weapon offers significant versatility in the field, capable of taking down both small and large targets. Standard clip size is 98 rounds."
+"asw_wdesc_rifle"		"Il 22A3-1 è l'arma di combattimento standard in dotazione ai soldati delle Forze Armate Interstellari (IAF). Grazie all'elevata cadenza di fuoco, ai proiettili esplosivi ad alta velocità e al lanciagranate sottocanna, quest'arma offre una notevole versatilità sul campo, in grado di abbattere bersagli piccoli e grandi. La dimensione standard del caricatore è di 98 colpi."
```
Comment (Steffo99) [inline resource/reactivedrop_italian.txt:2409]:
```suggestion
"asw_wdesc_rifle"		"Il 22A3-1 è l'arma di combattimento standard in dotazione ai soldati delle Forze Armate Interstellari (IAF). Grazie all'elevata cadenza di fuoco, ai proiettili esplosivi ad alta velocità e al lanciagranate sottocanna, quest'arma offre una notevole versatilità sul campo, essendo in grado di abbattere bersagli piccoli e grandi. La dimensione standard del caricatore è di 98 colpi."
```

**Review by Steffo99 — COMMENTED** [review]
This sounds a bit better to me!

File: resource/reactivedrop_koreana.txt [hunk 2]
```diff
@@ -2405,8 +2405,8 @@
...
 "asw_reserved_tt"		"해병이 이전 사령관을 위해 예약된 상태입니다."
-"[english]asw_wdesc_rifle"		"The 22A3-1 is the standard issue combat weapon given to an Interstellar Armed Forces marine. Offering both high rate of fire, high velocity explosive tipped rounds and an underslung grenade launcher, this weapon offers significant versatility in the field, capable of taking down both small and large targets. Standard clip size is 98 rounds."
-"asw_wdesc_rifle"		"22A3-1 돌격 소총은 성간 무장 부대의 해병들에게 기본으로 지급되는 무기입니다. 높은 연사력과 폭발물이 내장된 고속 탄약 그리고 유탄발사기까지 장착되어 있는 이 무기는 크고 작은 목표들을 손쉽게 제거할 수 있어 전장에서 뛰어난 활용성을 발휘합니다. 기본 탄창 크기는 98발입니다."
+"[english]asw_wdesc_rifle"		"The 22A3-1 is the standard issue combat weapon given to an Interstellar Armed Forces marine. Offering high rate of fire and high velocity explosive rounds for the underslung grenade launcher, this weapon offers significant versatility in the field, capable of taking down both small and large targets. Standard clip size is 98 rounds."
+"asw_wdesc_rifle"		"22A3-1 돌격 소총은 성간 무장 부대의 해병들에게 기본으로 지급되는 무기입니다. 이 무기는 빠른 발사 속도, 고속 폭발성 팁 탄과 언더슬링 유탄 발사기를 모두 갖추고 있어 작은 목표물과 큰 목표물을 모두 쓰러뜨릴 수 있어 현장에서 활용도가 매우 높습니다. 기본 탄창 크기는 98발입니다."
```
Comment (906bc906) [inline resource/reactivedrop_koreana.txt:2409]: needs to be changed, but I'd like to hear from other Korean translators about how to retranslate.

Please wait for suggestion from @hangumi 

```suggestion
"asw_wdesc_rifle"		"22A3-1 돌격 소총은 성간 무장 부대의 해병들에게 기본으로 지급되는 무기입니다. 이 무기는 높은 연사력과 언더배럴 유탄발사기의 고속 고폭탄을 통해 크고 작은 목표물을 모두 쓰러뜨릴 수 있어 전장에서의 범용성이 매우 우수합니다. 기본 탄창 크기는 98발입니다."
```

File: resource/reactivedrop_tchinese.txt [hunk 3]
```diff
@@ -2405,8 +2405,8 @@
 "asw_reserved"		"已保留"
 "[english]asw_reserved_tt"		"Marine currently reserved for previous commander."
 "asw_reserved_tt"		"該名角色目前已保留給上一名玩家。"
-"[english]asw_wdesc_rifle"		"The 22A3-1 is the standard issue combat weapon given to an Interstellar Armed Forces marine. Offering both high rate of fire, high velocity explosive tipped rounds and an underslung grenade launcher, this weapon offers significant versatility in the field, capable of taking down both small and large targets. Standard clip size is 98 rounds."
-"asw_wdesc_rifle"		"22A3-1突擊步槍是星際武裝部隊的標準作戰武器。武器同時具備高射速、高爆子彈以及裝備有下掛式榴彈發射器，因此這把武器在戰鬥中具有多種用途，能夠摧毀小型以及大型目標。其標準彈匣容量是98發。"
+"[english]asw_wdesc_rifle"		"The 22A3-1 is the standard issue combat weapon given to an Interstellar Armed Forces marine. Offering high rate of fire and high velocity explosive rounds for the underslung grenade launcher, this weapon offers significant versatility in the field, capable of taking down both small and large targets. Standard clip size is 98 rounds."
+"asw_wdesc_rifle"		"22A3-1突擊步槍是星際武裝部隊的標準作戰武器。這款武器提供高射速和高速爆炸彈，適用於下掛式榴彈發射器，在實戰中具有顯著的多功能性，能夠擊倒小型和大型目標。其標準彈匣容量是98發。"
```
Comment (finishtw) [inline resource/reactivedrop_tchinese.txt:2409]: This should be better
```suggestion
"asw_wdesc_rifle"		"22A3-1突擊步槍是星際武裝部隊的標準作戰武器。其具備高射速機能和下掛式榴彈發射器所使用的高爆彈藥，在戰場中適用於不同用途，能夠擊倒小型和大型目標。其標準彈匣容量是98發。"
```

File: resource/reactivedrop_finnish.txt [hunk 4]
```diff
@@ -2405,8 +2405,8 @@
...
-"asw_wdesc_rifle"		"22A3-1 on Tähtienvälisten Asevoimien jääkärin palveluskivääri. Suuri tulinopeus, suuren lähtönopeuden omaavat räjähdekärkiset ammukset ja piipun alla oleva kranaattiampumalaite mahdollistavat suuren monipuolisuuden kentällä, tehden mahdolliseksi sekä suurten että pienten kohteiden tuhoamisen. Lippaan koko on 98 laukausta."
+"[english]asw_wdesc_rifle"		"The 22A3-1 is the standard issue combat weapon given to an Interstellar Armed Forces marine. Offering high rate of fire and high velocity explosive rounds for the underslung grenade launcher, this weapon offers significant versatility in the field, capable of taking down both small and large targets. Standard clip size is 98 rounds."
+"asw_wdesc_rifle"		"22A3-1 on Tähtienvälisten Asevoimien jääkärin palveluskivääri. Tämä ase tarjoaa sekä suuren tulinopeuden, suuren nopeuden räjähdekärjellä varustetut ammukset että alhaalla olevan kranaatinheittimen, joten se tarjoaa merkittävää monipuolisuutta kentällä ja kykenee tuhoamaan sekä pieniä että suuria kohteita. Lippaan koko on 98 laukausta."
```
Comment (Crazymanbos) [inline resource/reactivedrop_finnish.txt:2409]: Looks good to me.

File: resource/reactivedrop_koreana.txt (hunk 2 above)
Comment (hangumi) [reply resource/reactivedrop_koreana.txt:2409]: > needs to be changed, but I'd like to hear from other Korean translators about how to retranslate.
> 
> Please wait for suggestion from @hangumi 
> 
> ```suggestion
> "asw_wdesc_rifle"		"22A3-1 돌격 소총은 성간 무장 부대의 해병들에게 기본으로 지급되는 무기입니다. 이 무기는 높은 연사력과 언더배럴 유탄발사기의 고속 고폭탄을 통해 크고 작은 목표물을 모두 쓰러뜨릴 수 있어 전장에서의 범용성이 매우 우수합니다. 기본 탄창 크기는 98발입니다."
> ```

언더배럴을 하단부착으로만 바꾸시면 괜찮을 것 같습니다.

**Review by Crazy3lf — APPROVED** [review]

File: resource/reactivedrop_koreana.txt (hunk 2 above)
Comment (906bc906) [reply resource/reactivedrop_koreana.txt:2409]: Looks good to me. @Duke-M-commits

Comment (906bc906) [reply resource/reactivedrop_koreana.txt:2409]: > 언더배럴을 하단부착으로만 바꾸시면 괜찮을 것 같습니다.

He suggested changing `언더배럴(underbarrel in Korean)` to `하단부착(underslung in Korean)`, and this text reflects that change. @Duke-M-commits 

```suggestion
"asw_wdesc_rifle"		"22A3-1 돌격 소총은 성간 무장 부대의 해병들에게 기본으로 지급되는 무기입니다. 이 무기는 높은 연사력과 하단부착 유탄발사기의 고속 고폭탄을 통해 크고 작은 목표물을 모두 쓰러뜨릴 수 있어 전장에서의 범용성이 매우 우수합니다. 기본 탄창 크기는 98발입니다."
```

File: resource/reactivedrop_japanese.txt [hunk 5]
```diff
@@ -2405,8 +2405,8 @@
 "asw_reserved"		"予約済"
 "[english]asw_reserved_tt"		"Marine currently reserved for previous commander."
 "asw_reserved_tt"		"さっきまで使用していたプレイヤーにより予約されている隊員。"
-"[english]asw_wdesc_rifle"		"The 22A3-1 is the standard issue combat weapon given to an Interstellar Armed Forces marine. Offering both high rate of fire, high velocity explosive tipped rounds and an underslung grenade launcher, this weapon offers significant versatility in the field, capable of taking down both small and large targets. Standard clip size is 98 rounds."
-"asw_wdesc_rifle"		"22A3-1アサルトライフルはIAFの海兵隊に支給される標準装備だ。 高い連射性能と弾速を持ち、銃身の下にはグレネードランチャーが付属している。 この武器は戦場での汎用性がとても高く、敵が小型でも大型でも相手にできる。 標準の弾倉サイズは98発入り。"
+"[english]asw_wdesc_rifle"		"The 22A3-1 is the standard issue combat weapon given to an Interstellar Armed Forces marine. Offering high rate of fire and high velocity explosive rounds for the underslung grenade launcher, this weapon offers significant versatility in the field, capable of taking down both small and large targets. Standard clip size is 98 rounds."
+"asw_wdesc_rifle"		"22A3-1アサルトライフルはIAFの海兵隊に支給される標準装備だ。 この武器は、高火力、高速の爆発性チップ弾と、下方に吊り下げられたグレネード・ランチャーの両方を備えており、小型のターゲットから大型のターゲットまで倒すことができる。 標準の弾倉サイズは98発入り。"
```
Comment (Deerlord0) [inline resource/reactivedrop_japanese.txt:2409]: "asw_wdesc_rifle"		"22A3-1アサルトライフルはIAFの海兵隊に支給される標準装備だ。 高い連射性能を持ち、銃身下に装着されたグレネードランチャーからは炸裂弾を高速で射出できる。 この武器は戦場での汎用性がとても高く、敵が小型でも大型でも相手にできる。 標準の弾倉サイズは98発入り。"

**Review by Deerlord0 — CHANGES_REQUESTED** [review]
sent a fix for new AR description.

**Review by Steffo99 — APPROVED** [review]

**Review by DucDuyGaming — CHANGES_REQUESTED** [review]
I think I forgot to submit the review, here you go

**Comment by Duke-M-commits** [comment]
I've indented lines which nobody found time to look at. 
@BenLubar I think we're good to go

**Pull request merged**